internal/services/notificationhub/notification_hub_resource.go

```diff
@@ -134,6 +134,32 @@ func resourceNotificationHub() *pluginsdk.Resource {
 				},
 			},
 
+			"browser_credential": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"subject": {
+							Type:         pluginsdk.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
+						},
+						"vapid_private_key": {
```

Comment:
This should probably be marked as `Sensitive`

Reply:
Fixed.